HeliPort/Appearance/StatusMenu.swift

```diff
@@ -151,6 +162,20 @@ final class StatusMenu: NSMenu, NSMenuDelegate {
     private let macItem = NSMenuItem(title: NSLocalizedString("Address: ", comment: "") + "(null)")
     private let itlwmVerItem = NSMenuItem(title: NSLocalizedString("Version: ", comment: "") + "(null)")
 
+    private let enableLoggingItem = NSMenuItem(title: NSLocalizedString("Enable Wi-Fi Logging", comment: ""))
+    private let createReportItem = NSMenuItem(title: NSLocalizedString("Create Diagnostics Report...", comment: ""))
+    private let diagnoseItem = NSMenuItem(title: NSLocalizedString("Open Wireless Diagnostics...", comment: ""))
+    private let hardwareInfoSeparator = NSMenuItem.separator()
+
+    private let manuallyJoinItem = NSMenuItem(title: NSLocalizedString("Join Other Network...", comment: ""))
+    private let createNetworkItem = NSMenuItem(title: NSLocalizedString("Create Network...", comment: ""))
+    private let networkPanelItem = NSMenuItem(title: NSLocalizedString("Open Network Preferences...", comment: ""))
+
+    private let aboutItem = NSMenuItem(title: NSLocalizedString("About HeliPort", comment: ""))
+    private let checkUpdateItem = NSMenuItem(title: NSLocalizedString("Check for Updates...", comment: ""))
+    private let quitItem = NSMenuItem(title: NSLocalizedString("Quit HeliPort", comment: ""),
```

Comment:
You don't need to declare the action for `quickItem` since you already declare it in lines 293

Reply:
It is needed for the constructor if initialized with `keyEquivalent`, or it will give a compile error, any idea?

Reply:
I missed that! Nvm, I mean anyways it will still add the action when it calls `addClickItem` lol.